types/types.go

```diff
@@ -302,3 +302,28 @@ type GetCgCountResp struct {
 type CommitInfoResp struct {
 	LastSuccessfulCommitId string `json:"commit_id"`
 }
+
+// ML Based Test Selection Request and Response
+type MLSelectTestsRequest struct {
+	MLServiceAPIRequest MLServiceAPIRequest `json:"ml_service_api_request"`
+	Percentile          int                 `json:"percentile"`
+}
+
+type MLServiceAPIRequest struct {
```

Comment:
This should go to types.go of ML client.

Reply:
Do you know how I can import it here using an scm url then?

Reply:
I'll keep it here and in ml-client code for now. Once ti-service changes are merged to github i'll remove it from here and use the struct in ml-client in the ml-ti-cli. Is that okay?

Reply:
Let's discuss this tomorrow.

Reply:
As discussed, keeping this here as well as in ML Client with a manual copy to ML Client